tests/core/cross/BUILD.bazel

```diff
@@ -33,6 +33,31 @@ go_binary(
     deps = [":platform_lib"],
 )
 
+go_binary(
+    name = "native_bin",
+    srcs = ["main.go"],
+    pure = "on",
```

Comment:
Is `pure` needed here? If yes, I think it should be at least documented.

Reply:
It is only required in this test so that there's no dependency on darwin/linux/windows cpp toolchains. However, for an arbitrary user of `go_cross` the binary is not required to be pure